### ISSUE TITLE
Error on publish

### ISSUE THREAD

**Issue by thepra** [LostBeard/SpawnDev.BlazorJS#42]
**Describe the bug**
>Microsoft.NET.Sdk.StaticWebAssets.Compression.targets(345,5): Error MSB4018 : errore imprevisto dell'attività "ApplyCompressionNegotiation".
System.InvalidOperationException: Endpoints not found for compressed asset: C:\<redacted>\obj\Release\net9.0\compressed\4t6f7o5fzg-j538s5lvtp.gz
   at Microsoft.AspNetCore.StaticWebAssets.Tasks.ApplyCompressionNegotiation.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(TaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask)

After adding the package to the blazor wasm solution I get this error at publishing to a folder. In my case the call to `.AddBlazorJSRuntime()` is within a components library meanwhile I call `.BlazorJSRunAsync()` in the usual `Program.cs` file as usual.

**To Reproduce**
1. Add the NuGet package to the solution
2. Click Publish to folder

**Expected behavior**
The publishing works as usual

**Desktop (please complete the following information):**
 - OS: Windows 10 x64
 - Browser edge

**Comment by LostBeard** [comment]
Thanks for reporting your issue! 

This issue has already been reported on my other repository for SpawnDev.BlazorJS.WebWorkers as [issue #4](https://github.com/LostBeard/SpawnDev.BlazorJS.WebWorkers/issues/4) and was determined to be a bug in .Net 9.

I reported the issue to Microsoft after I determined the issue is in their code: [ApplyCompressionNegotiation build task fails when RCL references a Nuget RCL with StaticWebAssetBasePath set to /](https://github.com/dotnet/aspnetcore/issues/59291)

The issue was originally set to be fixed in .Net 10 but Microsoft coders have since found the source of the issue and it should be fixed in a future release of .Net 9.
